### ISSUE TITLE
fix commented out CloseDatabase calls in export_db.py test

### ISSUE THREAD

**Issue by aowen87** [visit-dav/issues-test#2622]
I ran across this but didn't have time to debug. 

All the CloseDatabase calls are commented out in export_db.py test. It was an inadvertent commit.

But, if you un-comment them all out, the engine crashes in parallel.

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. The following information
could not be accurately captured in the new ticket:

Original author: Mark Miller
Original creation: 06/29/2016 05:57 pm
Original update: 07/12/2016 06:39 pm
Ticket number: 2638
